### ISSUE TITLE
Not receiving email for `pod trunk register`

### ISSUE THREAD

**Issue by scannillo** [CocoaPods/cocoapods-trunk#170]
Hello,

I am long-time CocoaPods publisher and am now no longer receiving an email after `pod trunk register` when using a new machine. Here is the command I am trying: 

```
pod trunk register scannillo@paypal.com 'Samantha Cannillo' --description='2nd Dev Intel Mac'
```

I greatly appreciate your help into getting this resolved! Please let me know if there is any additional information I can provide.

**Comment by scannillo** [comment]
An email was sent!

**Comment by scannillo** [comment]
Hello - this is not consistently working. `samantha.cannillo@getbraintree.com`, as well as other `@getbraintree.com` teammate emails are not receiving emails from `pod trunk register`.

**Comment by crookedneighbor** [comment]
My `blade@getbraintree.com` email is also not getting an email when I run the command.

**Comment by orta** [comment]
Strange, it looks like it was blocked on your side, either that or someone beat me to removing these from the blocklist in our mailer. I've seen a few `@getbraintree.com` emails go out since, so maybe a temporary blip?

 e.g. I'd retry

**Comment by scannillo** [comment]
Once we switched to our `@paypal.com` emails, we were able to get registered. I can close this ticket, but if you could unblock the `@getbraintree.com` domain that would be great!

**Comment by scannillo** [comment]
👋 Hello @orta - we are again having this issue receiving emails to `@paypal.com`. I create a new issue [here](https://github.com/CocoaPods/CocoaPods/issues/11850), but if you're able to help us out that would be greatly appreciated!
